packages/stripe_android/android/src/main/kotlin/com/reactnativestripesdk/PaymentOptionDisplayDataMapper.kt

```
@@ -164,12 +164,12 @@ private fun getHtmlTagsForSpanStyle(spanStyle: SpanStyle): List<Pair<String, Str
   }
 
   spanStyle.color.takeIf { it != Color.Unspecified }?.let { color ->
-    val hexColor = String.format("#%06X", 0xFFFFFF and color.toArgb())
+    val hexColor = String.format("#%06X", RGB_MASK and color.toArgb())
```

detekt (1.23.8)

[warning] String.format("#%06X", RGB_MASK and color.toArgb()) uses implicitly default locale for string formatting.

(detekt.potential-bugs.ImplicitDefaultLocale)